kitsune/users/tests/test_templates.py

```diff
@@ -446,6 +446,15 @@ def test_bio_links_nofollow(self):
         doc = pq(r.content)
         eq_(2, len(doc('.bio a[rel="nofollow"]')))
 
+    def test_bio_links_no_img(self):
+        # bug 1427813
+        self.profile.bio = 'my image <img src="https://www.example.com/the-dude.jpg">'
+        self.profile.save()
+        r = self.client.get(reverse('users.profile', args=[self.u.username]))
+        eq_(200, r.status_code)
+        doc = pq(r.content)
+        eq_(0, len(doc('.bio img')))
```

Comment:
In cases like this I believe checking against the exact final text is safer.

Reply:
I considered that but that does not seem to be how it works now. It looks like we store the raw input and wiki process it at render time. I usually like to do rendering and cleaning on update then just serve it straight out of the DB but that would be a good bit more work.